rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-10-05
\ No newline at end of file
+nightly-2022-01-10
```

Comment:
Do we need this? how does this work specifying a future `2022` version?

Reply:
Nah, it was a mistake